src/main/kotlin/commands/RemoveVersion.kt

```diff
@@ -22,22 +22,25 @@ import token.TokenStore
 import utils.GitHubUtils
 
 class RemoveVersion : CliktCommand(name = "remove") {
-    private val identifierParam: String? by option("--id", "--package-identifier")
+    private val isCIEnvironment = System.getenv("CI")?.toBooleanStrictOrNull() == true
+    private val packageIdentifierParam: String? by option("--id", "--package-identifier")
+    private val packageVersionParam: String? by option("--version", "--package-version")
+    private val reasonForDeletionParam: String? by option("--reason", "--reason-for-deletion")
```

Comment:
It's a given that these are parameters so they don't need `Param` in the variable name.